bin/reth/src/args/multi_use_args.rs

```diff
@@ -0,0 +1,49 @@
+use crate::dirs::{PlatformPath, SecretKeyPath};
+use hex::encode as hex_encode;
```

Comment:
I expect that we will get more "multi_use_arg" types.

but this file also doesn't define args, I'd like to rename this to `secret_key` instead